### DISCUSSION TITLE
error- Error HH600: Compilation failed

### DISCUSSION THREAD

**Discussion by TimeKeyRoll** [NomicFoundation/hardhat#3019] [Troubleshooting] [answered]
Hi! just trying to compile but this error won't go. any help would be appreciated.
![L-7 error of compile](https://user-images.githubusercontent.com/101450342/183279568-0df0282e-baef-4972-a0db-a2720206a181.PNG)

**Comment by fvictorio** [comment, marked as answer]
`ParserError` means you have a syntax error in your contract. This is related to how you are using Solidity, not to Hardhat itself.
